cookbooks/aws-parallelcluster-config/resources/fetch_config.rb

```diff
@@ -36,11 +45,17 @@ def fetch_cluster_config(config_path)
                          " --region #{node['cluster']['region']}" \
                          " #{config_path}"
     fetch_config_command += " --version-id #{node['cluster']['cluster_config_version']}" unless node['cluster']['cluster_config_version'].nil?
-    execute "copy_cluster_config_from_s3" do
-      command fetch_config_command
-      retries 3
-      retry_delay 5
-    end
+    execute_command("copy_cluster_config_from_s3", fetch_config_command)
+  end
+
+  def fetch_change_set
```

Comment:
Can this function be merged with `fetch_cluster_config`?

Reply:
not sure I get the question correctly, but the two have different scope. Furthermore the cluster config is always fetched while the change set is fetched only when queues params are updated.

Reply:
done